edx_proctoring/views.py

```diff
@@ -228,7 +271,18 @@ def get(self, request, course_id, content_id=None):
         is_learning_mfe = request.GET.get('is_learning_mfe') in ['1', 'true', 'True']
         content_id = request.GET.get('content_id', content_id)
 
-        active_exams = get_active_exams_for_user(request.user.id)
+        user_id = request.user.id
+        requested_user_id = request.GET.get('user_id', None)
+        if requested_user_id:
+            if request.user.is_staff:
```

Comment:
Just checking my own understanding, but a user can never supply the `requested_user_id` value via some edx-exams flow. And the edx-exams service worker is staff, so it will still get the correct learner data.

Reply:
yeah the user_id passed in here is determined by edx exams and not any user input.